task1/TaskMethods.cs

```diff
@@ -0,0 +1,361 @@
+                    if (secondNum == 0)
+                    {
+                        Console.WriteLine("Can't divide by zero");
+                        break;
+                    }
+                    result = Div(firstNum, secondNum);
+                    break;
+                default:
+                    break;
+            }
+            return result;
+        }
+
+        private static double Div(double firstNum, double secondNum)
+        {
+            return firstNum / secondNum;
+        }
+
+        private static double Mul(double firstNum, double secondNum)
+        {
+            return firstNum * secondNum;
+        }
+
+        private static double Sub(double firstNum, double secondNum)
+        {
+            return firstNum - secondNum;
+        }
+
+        private static double Add(double firstNum, double secondNum)
+        {
+            return firstNum + secondNum;
+        }
+
+        //
+        // Currency converter
+        //
+        public static void ConvertCurrency()
+        {
+            decimal amount;
+            Console.WriteLine("Input amount of money");
+            Decimal.TryParse(Console.ReadLine(), out amount);
+            Console.WriteLine("Input currency");
+            string currency = Console.ReadLine();
+            decimal result = PerformConvertation(amount, currency);
+            Report(amount, result, currency);
+        }
+
+        private static decimal PerformConvertation(decimal amount, string currency)
+        {
+            decimal result = 0.00m;
+            switch (currency)
+            {
+                case "dollar":
+                    result = Convert.ToDecimal(Convert.ToDouble(amount) * 2.777);
+                    break;
+                case "euro":
+                    result = Convert.ToDecimal(Convert.ToDouble(amount) * 3.125);
+                    break;
+                case "pound":
+                    result = Convert.ToDecimal(Convert.ToDouble(amount) * 5.421);
+                    break;
+                default:
+                    break;
+            }
+            return result;
+        }
+
+        private static void Report(decimal amount, decimal result, string currency)
+        {
+            Console.WriteLine($"Amount of money {amount} is {result} {currency}");
+        }
+
+        //
+        // Credit.
+        //
+        public static void GetCreditCondition()
+        {
+            decimal credit = 700;
+            Console.WriteLine("Input amount of payment");
+            Decimal.TryParse(Console.ReadLine(), out decimal amount);
+            if (amount < 100)
+            {
+                Console.WriteLine("Payment must be 100 minimum");
+                return;
+            }
+            decimal rest = CreditCalculation(credit, amount);
+            AnalyzeOfRest(rest, credit);
+        }
+
+        private static void AnalyzeOfRest(decimal rest, decimal credit)
+        {
+            if (rest > 0)
```

Comment:
There is a constant value for zero in decimal, please use it everywhere for decimals:
decimal.Zero